### PR TITLE
Add Polish translations for Jenkins

### PR DESCRIPTION
**Pull request by damianszczepanik**
Adds more translations for Polish, fixes some typos

### Testing done

Tested locally on version 2.361.2

### Submitter checklist

- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@jenkinsci/core-pr-reviewers 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7325"></a>

### CODE REVIEW

File: core/src/main/resources/hudson/model/Messages_pl.properties
Comment (DuMaM) [inline core/src/main/resources/hudson/model/Messages_pl.properties:36]: Hi,
I'm wondering why did you changed this to lowercase? 
Also this class refers to all kind of items, so maybe instead of `project` we should use something closer to `item` like `obiekt` or `element`? 
What do you think?

File: core/src/main/resources/hudson/widgets/HistoryWidget/index_pl.properties
Comment (DuMaM) [inline core/src/main/resources/hudson/widgets/HistoryWidget/index_pl.properties:25]: should this be lowercase?

File: core/src/main/resources/jenkins/diagnostics/ControllerExecutorsNoAgents/message_pl.properties
Comment (DuMaM) [inline core/src/main/resources/jenkins/diagnostics/ControllerExecutorsNoAgents/message_pl.properties:23]:
```suggestion
ExecutorsWarning=Uruchamianie zadań na głównym węźle (kontrolerze) może prowadzić do problemów z bezpieczeństwem. \
  Zalecane jest skonfigurowane agentów, do uruchamiania zadań poza kontrolerem. \
  Sprawdź <a href="https://www.jenkins.io/redirect/building-on-controller/" target="_blank" rel="noopener noreferrer">dokumentację</a>.
```

File: core/src/main/resources/lib/hudson/project/config-trigger_pl.properties
Comment (DuMaM) [inline core/src/main/resources/lib/hudson/project/config-trigger_pl.properties:22]: I think old translation was better. 
Polish word which refers to `Build` is used here more like a `task`.
Jobs not always build stuff. 😉 
```suggestion
Build\ Triggers=Wyzwalacze zadania
```

**Review by DuMaM — CHANGES_REQUESTED** [review]
Hi,
thanks for this PR 😉 
I proposed some changes. Let me know what you think about it.
Also I would be grateful if you can join forces with me to provide Polish translations for plugins in crowdin service.
https://crowdin.jenkins.io/

File: core/src/main/resources/hudson/model/Messages_pl.properties
Comment (damianszczepanik) [reply core/src/main/resources/hudson/model/Messages_pl.properties:36]: There are two items here: line 35 with upper case and 36 with lower.
35 -> `Project` myFooProject
36 -> Delete `project`

and what I have observed there is no need to introduce new translation for `project` but that would be good idea for `view` or `build`. IMO having placeholder like `Delete {}` that is used for any object/item does not help to make good localizations :)

File: core/src/main/resources/lib/hudson/project/config-trigger_pl.properties
Comment (damianszczepanik) [reply core/src/main/resources/lib/hudson/project/config-trigger_pl.properties:22]: Agree with your but then we should decide if we want to have better translation OR consistent translation:
obraz
Build Steps -> Kroki zadania - IMO it sounds strange and we should keep original concept even this is not perfect

File: core/src/main/resources/hudson/widgets/HistoryWidget/index_pl.properties
Comment (damianszczepanik) [reply core/src/main/resources/hudson/widgets/HistoryWidget/index_pl.properties:25]: Honestly I have no idea as I haven't seen this action. I decided to fix typo which I found in repository :)
So maybe not perfect but still better than it was!

Comment (DuMaM) [reply core/src/main/resources/hudson/widgets/HistoryWidget/index_pl.properties:25]: Sure :)

File: core/src/main/resources/lib/hudson/project/config-trigger_pl.properties
Comment (DuMaM) [reply core/src/main/resources/lib/hudson/project/config-trigger_pl.properties:22]: Make sense, i wasnt able to find better translation.
Maybe `Składowe zadania`
when you have a math problem to solve there are steps you need to execute.
then you can refer to them as `Kroki zadania`

File: core/src/main/resources/hudson/model/Messages_pl.properties
Comment (DuMaM) [reply core/src/main/resources/hudson/model/Messages_pl.properties:36]: sad.

File: core/src/main/resources/lib/hudson/project/config-trigger_pl.properties
Comment (damianszczepanik) [reply core/src/main/resources/lib/hudson/project/config-trigger_pl.properties:22]: Well, if you ask your dev team, tester and business consultant (all of then use Jenkins) about "Składowe zadania" without context and 4 of 5 give you good answer what is "Składowe zadanie" we can go for it and change it. But I believe they wont' give you that answer :)
Also mind that they know what is build because they have already used this word for TeamCity, Travis and other tools. 
_\- Can you start the build Mike?_ (sure he will and he knows that you are talking about CI/CD)
_\- Can you start the task Tom?_ (what task, what tool? probably JIRA subtask)

We should use same names no matter what language we speak. That helps us to understand each other :)

Comment (DuMaM) [reply core/src/main/resources/lib/hudson/project/config-trigger_pl.properties:22]: Ok, i'm sold 😉

**Review by DuMaM — APPROVED** [review]

**Comment by DuMaM** [comment]
One more time thanks for this update. 
I would be grateful if you help me with translations for plugins in https://crowdin.jenkins.io/

**Review by NotMyFault — APPROVED** [review]
Tested with the locale plugin, looks good so far.

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
